pkg/cryptoauthlib/Makefile

```diff
@@ -1,6 +1,6 @@
 PKG_NAME=cryptoauthlib
 PKG_URL=https://github.com/MicrochipTech/cryptoauthlib
-PKG_VERSION=af8187776cd3f3faf8bed412eaf6ff7221862e19
+PKG_VERSION=9a37b8d685c351faed720d18d9610345bdaf20b0
```

Comment:
What do you think about checking out a release?

Reply:
How can I do that? Add the tag instead of the commit number?

Reply:
I think that should be possible, yes. Otherwise, use that commit hash.

Reply:
Hmm, I like stating SHA1. Tags could be relocated - even though it's considered to be evil ;-)